src/vs/workbench/contrib/terminal/common/terminalConfiguration.ts

```diff
@@ -78,7 +78,7 @@ const terminalConfiguration: IConfigurationNode = {
 		},
 		[TerminalSettingId.DefaultLocation]: {
 			type: 'string',
-			enum: [TerminalLocation.Editor, TerminalLocation.TerminalView],
+			enum: [TerminalLocation.Editor, TerminalLocation.Panel],
```

Comment:
We need to keep these are strings to make them friendly settings, map it back to 0/1 when you access it